tests/modeladmin/tests.py

```diff
@@ -298,6 +298,21 @@ class BandAdmin(ModelAdmin):
             list(list(ma.get_formsets_with_inlines(request))[0][0]().forms[0].fields),
             ['main_band', 'opening_band', 'day', 'id', 'DELETE'])
 
+    def test_overriding_get_exclude(self):
+        """
+        Ensure that overriding ModelAdmin.get_exclude method works
+        Refs #24941.
+        """
+        class BandAdmin(ModelAdmin):
+            exclude = ['bio']
+
+            def get_exclude(self, *args, **kwargs):
```

Comment:
I'd use the signature from the documentation. Also, a more complex example that uses `obj` might be nice. This simple test doesn't give me great confidence that there aren't any edge cases that might cause problems. Are the changes in `get_formset()` covered by this test?